### PR TITLE
Removes reportertron from round start

### PR DESCRIPTION
**Pull request by JamieD1**
Replaces Reporter with paladin

:cl:  
tweak: Replaces Reporter with paladin
/:cl:

### CODE REVIEW

**Comment by Xantam-13** [comment]
Replace the turd with turd+sugar - good improvement! 

Best of a bad bunch

**Comment by maxion12345** [comment]
what the fuck is wrong with you

**Comment by Runian** [comment]
Why the change?

**Comment by maxion12345** [comment]
clarifying on my earlier comment because it's probably just another "good headcoders follow orders" moment but paladin was removed from round start rotation for good reason; it was validhunt incarnate, spawning an entire gang of ai mains who existed to simp for security and cuck every single antagonist out there under the justification of it's last law. After all, how could punishing people who harm innocents be considered dishonorable or evil? Add in a law rewiring them to respect station authority and simp for command/sec above anyone else and you have a wonderful mix for validhunting.

**Comment by Smudged** [comment]
Why add a valid-hunting law-set to the round start pool in the first place. 

At least Reporter is interesting.

With this change you're really only giving an AI the chance to be able to valid hunt antags round start.

File: config/game_options.txt [hunk 1]
```diff
@@ -441,7 +441,7 @@ LAW_WEIGHT crewsimov,5
 ION_LAW_WEIGHT crewsimov,1
 LAW_WEIGHT asimovpp,0
 ION_LAW_WEIGHT asimovpp,2
-LAW_WEIGHT paladin,0
+LAW_WEIGHT paladin,3
```
Comment (JamieD1) [inline config/game_options.txt]:
```suggestion
LAW_WEIGHT paladin,0
```

**Pull request merged**